src/calendar.rs

```diff
@@ -105,6 +105,15 @@ impl ToString for Calendar {
     }
 }
 
+impl std::convert::TryInto<String> for &Calendar {
```

Comment:
This should not be needed as this is redundant with `impl ToString for Calendar {` just above. See https://github.com/hoodie/icalendar-rs/issues/11#issuecomment-573326072